UIforETW/Utility.cpp

```diff
@@ -349,78 +558,83 @@ int DeleteFiles(HWND hwnd, const std::vector<std::wstring>& paths)
 		FO_DELETE,
 		&fileNames[0],
 		NULL,
-		FOF_ALLOWUNDO | FOF_FILESONLY | FOF_NOCONFIRMATION,
+		(FOF_ALLOWUNDO | FOF_FILESONLY | FOF_NOCONFIRMATION),
 	};
 	// Delete using the recycle bin.
-	int result = SHFileOperation(&fileOp);
-
-	return result;
+	//TODO: IFileOperation?
+	return ::SHFileOperationW(&fileOp);
 }
 
 void SetClipboardText(const std::wstring& text)
 {
-	BOOL cb = OpenClipboard(GetDesktopWindow());
-	if (!cb)
+	ATLVERIFY(::OpenClipboard(::GetDesktopWindow()));

```

Comment:
Two basically irrelevant comments, but I want some of this useless knowledge to be useful:
1) From following the VsVim project, OpenClipboard can definitely fail in relatively normal scenarios.
2) GlobalAlloc(GMEM_MOVEABLE, ...) failing isn't quite as "should actually never happen" as with normal malloc -- there's a limit of about 64k outstanding moveable allocations. Given the expected frequency of that category of allocations, that's still well within the something is terribly wrong category if it fails, but it's not quite as dramatic as with normal address space exhaustion.